src/rdkafka_conf.c

```diff
@@ -1330,8 +1330,8 @@ rd_kafka_anyconf_set_prop (int scope, void *conf,
 
 			/* No match */
 			rd_snprintf(errstr, errstr_size,
-				 "Invalid value for "
-				 "configuration property \"%s\"", prop->name);
+				 "Invalid value \"%.*s\" for "
+				 "configuration property \"%s\"", (int)(t-s), s, prop->name);
```

Comment:
Move the three arguments after `..configuration property..", ` to a new line to keep < 80 column width.